tests/ui/cypress_specs/multi_user_config_file_with_ta.js

```diff
@@ -238,13 +241,18 @@ describe('Config File Users with TA', () => {
 
   add_roa_test_settings.forEach(function (ts) {
     it('Add ROA for CA ' + ts.ca + ' as ' + ts.d + ' user should ' + (ts.o ? 'succeed' : 'fail'), () => {
+      cy.intercept('GET', '/api/v1/cas/' + ts.ca + '/routes/analysis/full').as('analyzeRoutes')
```

Comment:
This change is actually for #377 but was made here because #377 was blocking testing for #371.